bin/template/cordova/build

```diff
@@ -19,15 +19,21 @@
     under the License.
 */
 
-var build = require('./lib/build'),
-    args  = process.argv;
+const build = require('./lib/build');
+const args = process.argv;
 
 // provide help
-if ( args[2] == '--help' || args[2] == '/?' || args[2] == '-h' || args[2] == '/h' ||
-                    args[2] == 'help' || args[2] == '-help' || args[2] == '/help') {
+if (
```

Comment:
```
if (['--help', '/?', '-h', '/h', 'help', '-help', '/help'].indexOf(args[2]) > -1 ) {
 ... // to tricky?
}
```

Reply:
I can update this. I didn't refactor every line so this was left from the previous release.

https://github.com/apache/cordova-electron/blob/3528b1dbbc5cc01308e5d0f0626ac1852c95c437/bin/template/cordova/build#L27